tests/integration/conftest.py

```diff
@@ -804,16 +809,19 @@ def _remove_client_tools_repo_oracle(shell):
 
 
 @pytest.fixture
-def satellite_registration(shell, yum_conf_exclude):
+def satellite_registration(shell, yum_conf_exclude, request):
     """
     Register the system to the Satellite server
     """
-    if "oracle" in SYSTEM_RELEASE_ENV:
-        yum_conf_exclude(exclude=["rhn-client*"])
-    _add_client_tools_repo_oracle(shell)
     # Get the registration command for the respective system
     # from the conftest function
     sat_reg_command = satellite_registration_command()
+    # If the fixture is parametrized, use the parameter as the registration command
+    if hasattr(request, "param"):
+        sat_reg_command = request.param
```

Comment:
Would be worth it to add a print statement mentioning that the sat_reg_command is now from a param instead of coming from `satellite_registration_command()`? 

I think this could help us to understand if the value was changed somehow and is different from what we expect.

Reply:
That makes sense, thank you @r0x0d